README.md

```diff
@@ -1 +1,56 @@
-# AlumLink
\ No newline at end of file
+# AlumLink
+Development Setup
+Each user will need to do this on their local machine.
+
+Ensure you have PostgreSQL installed
+Check if you have PostgreSQL installed
+✅ versions 10-14 should work
+🚫 version 15 has not been tested
+If you need to install PostgreSQL see the installing PostgreSQL guides
+Create a PostgreSQL user and database
+The project-starter template expects the following for local development:
+
+PostgreSQL User/Role
+name: ctp_user
+password: ctp_pass
+PostgreSQL Database
+name: alumlinkdb
+For Windows/pgAdmin users
+If you are on Windows and installed pgAdmin follow our pgAdmin guide to create a user in PostgreSQL named ctp_user with the password ctp_pass and a database named alumlinkdb.
+
+For Mac/Linux users
+Create a user in PostgreSQL named ctp_user with the password ctp_pass:
+
+This only needs to be done one time on your machine You can create additional users if you want to.
+
+createuser -P -s -e ctp_user
```

Comment:
when you're showing code, use code snippets

if it's inline use backticks `

if it's a code snippet, surround it with 3 backticks ```